src/test/java/PlayingCardUnitTests.java

```diff
@@ -0,0 +1,49 @@
+import PlayingCard.Card;
+import PlayingCard.CardSuit;
+import PlayingCard.CardValue;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class PlayingCardUnitTests {
+
+    @Test
+    @DisplayName("Check if multiple of the same card is found in deck")
+    public void multipleInDeckCheck() {
+        var testDeck = Main.createDeck();
+        List<String> inDeck = new ArrayList<>();
+        for (Card card : testDeck) {
+            if (inDeck.contains("" + card.getSuit() + card.getValue())) {
+                Assertions.fail("Multiple of the same card found in deck");
+            } else {
+                inDeck.add("" + card.getSuit() + card.getValue());
+            }
+        }
+    }
+
+    @Test
+    @DisplayName("Check all correct cards are in deck")
```

Comment:
I see what you were thinking here. Asserting the negative outcome is however an anti-pattern. This test is successful when nothing is asserted. Something like this:
```
    @Test
    @DisplayName("A card will return a name when getName is called on a card constructed with a given a suit and value")
    public void CardWillReturnNameWhenConstructedWithSuitAndValue() {
        for (CardSuit suit : CardSuit.values()) {
            for (CardValue value : CardValue.values()) {
                var card = new Card(suit, value);
                var expectedName = "The " + value.name() + " of " + suit.name();

                Assertions.assertEquals(expectedName, card.getName());
            }
        }
    }

Reply:
I thought my logic on that one was a bit weird but thank you. I can see how I was testing main there with the creating the decks and not exclusively testing Card only which makes sense now

Reply:
Would you say there are any more tests that I should look to add to Card?

Reply:
You could add two more, one for getValue and one for getName that simply create a card and assert the functions return what is expected. But it's not necessary as getName uses both. Up to you.